src/pudl/metadata/resources/pudl.py

```diff
@@ -56,6 +56,8 @@
         "sources": ["pudl"],
     },
     "core_pudl__assn_utilities_plants": {
+        "title": "PUDL Utility-Plant Associations",
+        "description": "Associations between PUDL utility IDs and PUDL plant IDs. This table is read in from a spreadsheet stored in the PUDL repository: src/pudl/package_data/glue/pudl_id_mapping.xlsx",
```

Comment:
A few tables that I added titles and descriptions to before realizing there were dozens, an implemented the hacky workaround.